Tests/SwiftFormatPrettyPrintTests/AsExprTests.swift

```diff
@@ -1,5 +1,9 @@
+import XCTest
+
 final class AsExprTests: PrettyPrintTestCase {
-  func testWithoutPunctuation() {
+  func testWithoutPunctuation() throws {
+    throw XCTSkip("As expression grouping does not account for new sequence expression structure.")
```

Comment:
Let's wait for https://github.com/apple/swift-syntax/pull/619 to be merged so that we can handle all of these properly without major formatting regressions.

Reply:
To be clear, this has already regressed regardless of parsing backend as of https://github.com/apple/swift/commit/ed34dfc637732c600f9143fde6cee56a79848e9e

Reply:
Yes, you're right. We _could_ fix that by lifting swift-format's operating folding implementation into SwiftFormatCore instead of SwiftFormatPrettyPrint but I don't think it's worth the churn since https://github.com/apple/swift-syntax/pull/619 is on the horizon.

Ok, once you've made the change you mentioned in the other comment thread, we can land this and then do follow-ups.

Reply:
Very much agreed. Excited to be able to have a common operator folding library available for y'all. Thanks Tony.